src/lavinmqctl.cr

```diff
@@ -306,6 +306,23 @@ class LavinMQCtl
     URI.encode_www_form(@options["vhost"])
   end
 
+  MAX_PAGE_SIZE = 10_000
+
+  private def get(url, page = 1, items = [] of JSON::Any)
+    resp = http.get("#{url}?page=#{page}&page_size=#{MAX_PAGE_SIZE}", @headers)
+    handle_response(resp, 200)
+    if data = JSON.parse(resp.body).as_h?
+      items += data["items"].as_a
+      page = data["page"].as_i
+      if page < data["page_count"].as_i
+        return get(url, page + 1, items)
+      end
+    else
+      abort "invalid data"
```

Comment:
maybe a more detailed error message could be helpful here

`abort "Unexpected response from #{url}\n#{resp.body}"`